src/sql/workbench/services/connection/browser/passwordChangeDialog.ts

```diff
@@ -0,0 +1,162 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import 'vs/css!./media/passwordDialog';
+import { Button } from 'sql/base/browser/ui/button/button';
+import { Modal } from 'sql/workbench/browser/modal/modal';
+import { IConnectionProfile } from 'sql/platform/connection/common/interfaces';
+import { attachInputBoxStyler, attachCheckboxStyler } from 'sql/platform/theme/common/styler';
+import { INewConnectionParams } from 'sql/platform/connection/common/connectionManagement';
+import { IThemeService } from 'vs/platform/theme/common/themeService';
+import { IContextKeyService } from 'vs/platform/contextkey/common/contextkey';
+import { IClipboardService } from 'vs/platform/clipboard/common/clipboardService';
+import { localize } from 'vs/nls';
+import { InputBox } from 'sql/base/browser/ui/inputBox/inputBox';
+import { Checkbox } from 'sql/base/browser/ui/checkbox/checkbox';
+import { attachButtonStyler } from 'vs/platform/theme/common/styler';
+import { IContextViewService } from 'vs/platform/contextview/browser/contextView';
+import * as DOM from 'vs/base/browser/dom';
+import { ILogService } from 'vs/platform/log/common/log';
+import { IAdsTelemetryService } from 'sql/platform/telemetry/common/telemetry';
+import { IConnectionDialogService } from 'sql/workbench/services/connection/common/connectionDialogService';
+import { attachModalDialogStyler } from 'sql/workbench/common/styler';
+import { ILayoutService } from 'vs/platform/layout/browser/layoutService';
+import { ITextResourcePropertiesService } from 'vs/editor/common/services/textResourceConfiguration';
+import { LoadingSpinner } from 'sql/base/browser/ui/loadingSpinner/loadingSpinner';
+
+
+const OkText: string = localize('passwordChangeDialog.ok', "OK");
+const CancelText: string = localize('passwordChangeDialog.cancel', "Cancel");
+const DialogTitle: string = localize('passwordChangeDialog.title', "Change Password");
+const TitleIconClass: string = 'icon filterLabel';
+const newPasswordText: string = localize('passwordChangeDialog.newPassword', 'New password for SQL Server Login:');
+const confirmPasswordText: string = localize('passwordChangeDialog.confirmPassword', 'Confirm password:');
+const connectCheckboxText: string = localize('passwordChangeDialog.connectText', 'Connect?:');
```

Comment:
I don't think the label should have a `?` in the text.  Also, since this is for a checkbox that already has a label I don't think we need this label in the UI.